web/src/beta/components/Modal/index.tsx

```diff
@@ -1,54 +1,163 @@
-import { ReactNode } from "react";
+import { ReactNode, useState } from "react";
 
 import Wrapper from "@reearth/beta/components/Modal/ModalFrame";
-import Text from "@reearth/beta/components/Text";
-import { styled, useTheme } from "@reearth/services/theme";
+import { styled } from "@reearth/services/theme";
+
+type Size = "sm" | "md" | "lg";
+
+export type SidebarTab = {
+  id: string;
+  label: string;
+  content: ReactNode;
+  tabButton1?: ReactNode;
+  tabButton2?: ReactNode;
+};
 
 type Props = {
   className?: string;
-  title?: string;
-  size?: "sm" | "md" | "lg";
+  modalTitle?: string;
```

Comment:
This is a modal component, with only a single title prop. `title` is enough I think

Reply:
i keep it modalTitle cause some times the content will have Title in any further implementation for content title , but I will modified it